src/webview/components/Molecules/StatusIndicator/StatusIndicator.css

```diff
@@ -0,0 +1,87 @@
+:root {
+  background-color: var(--vscode-editor-background);
```

Comment:
we don't need to specific the background colour here since its already on the vscode editor background. only place we really need to use it is for somewhere where we need to hide something